default/stringtables/en.txt

```diff
@@ -1384,6 +1384,12 @@ Set the minimum number of human players required to start a multiplayer game.
 OPTIONS_DB_MP_HUMAN_MAX
 Limit the number of human players allowed in a multiplayer game. Set to -1 for unlimited.
 
+OPTIONS_DB_MP_CONN_HUMAN_MIN
+Minimum number of active human players, below which the server will stop the game being played.
+
+OPTIONS_DB_MP_DISCONN_HUMAN_MAX
+Maximum number of players, who control non-eliminated empires, who may disconnect from the server before it stops the game being played. Disabled if 0, meaning the server will never stop the game due to empire-playing players disconnecting.
```

Comment:
Is this actually the number of times a player can disconnect, ignoring any reconnect, or the number of empires that must have disconnected players, at the same time, for the server to abort?

ie. if a player disconnects, reconnects, disconnects, and reconnects, is that 2 or 0 disconnects / disconnected players?

Reply:
It's a number of empires that have disconnected players, at the same time.

Reply:
so then `disconnect` -> `be disconnected` and `disconnecting` -> `being disconnected`

Reply:
Fixed.